modules/android/pages/vector-search-api-reference.adoc

```diff
@@ -0,0 +1,286 @@
+= Vector Search API Reference
+:page-aliases: 
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:description: API reference for Vector Search in Couchbase Lite.
```

Comment:
Don't forget to expand this and include it in the body of the page with: 

[abstract]
{description}